packer/scripts/dep.sh

```diff
@@ -9,6 +9,9 @@ pip install --upgrade setuptools
 pip install ansible==2.2.0.0
 
 # set a friendly hostname
+# by default, the hostname in both /etc/hosts & /etc/hostname are obtained from DHCP server during install.
 rm -f /etc/hostname
 echo "rackhd" > /etc/hostname
-
+#Below two lines were moved from scripts/cleanup.sh. I can't recall why they didn't go with /etc/hostname change together...
+NEW_HOST_NAME=$(cat /etc/hostname)
+sed -i  "/127.0.1.1/,/$/c127.0.1.1\t${NEW_HOST_NAME}" /etc/hosts
```

Comment:
@yyscamper , of coz no.

typically, the /etc/hosts will be looked like below, the 127.0.1.1 is what we want to modify.
```
cat /etc/hosts
127.0.0.1       localhost
127.0.1.1       <Your_Machine_Name>

# The following lines are desirable for IPv6 capable hosts
::1     localhost ip6-localhost ip6-loopback
ff02::1 ip6-allnodes
ff02::2 ip6-allrouters

```